PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD001.xml

```diff
@@ -0,0 +1,201 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="MD001" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Ṣoma Dǝggʷā</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="AY"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt><p>Digitized within the framework of the PhD project "Scribes and Practices of
+                Manuscript Production in 21st Century Gondar Area, Ethiopia", under the project
+                of "Documenting an Ancient Education System in Africa: ʾAbənnat Təmhərt in
+                Ethiopia" funded by Gerda Henkel Stiftung. </p></editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1014MD"/>
+                        <idno>MD001</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="126v">1-126</locus>
+                            <title type="complete" ref="LIT3178Deggwa"/> 
+                            <textLang xml:lang="gez"></textLang>
+                            <colophon xml:id="coloph1"><locus target="#1r"></locus>ናሁ፡ ወጠንኩ፡ ጽሒፈ፡ ጾመ፡ ድጓ፡ በሥምረተ፡ እግዚአብሔር፡
+                                በዘመነ፡ ቅዱስ፡ ዮሐንስ፡ ወንጌላዊ፡ <sic>በ፲፻፡ ወ፱፻፡ ወ ፵፡ </sic> ዓመተ፡ ምህረት፡ ለሥምረተ፡ እግዚአብሔር፡ አሜን፡ ወአሜን፡ ለይኩን፡ ለይኩን፡ 
```

Comment:
Why is the year marked with `<sic>`?

Reply:
> Why is the year marked with `<sic>`?

I think it should written as ፲፱፻፵ or ፲፱፻ወ፵.